x/service/types/relay.go

```diff
@@ -25,7 +17,7 @@ func (relay *Relay) GetHash() ([32]byte, error) {
 		return [32]byte{}, err
```

Comment:
I know we started using `sha256.Size` instead of `32` in some places.  I think searching for `[32]` in the codebase will help capture all of these instances.